### ISSUE TITLE
"kubectl run nginx --image=nginx" results in error: no matches for kind "Deployment" in version "apps/v1beta1"

### ISSUE THREAD

**Issue by Dzhuneyt** [jamesward/kubernetes-the-easy-way#2]
```
❯ kubectl run nginx --image=nginx

WARNING: New generator "deployment/apps.v1beta1" specified, but it isn't available. Falling back to "run/v1".
kubectl run --generator=deployment/apps.v1beta1 is DEPRECATED and will be removed in a future version. Use kubectl create instead.
error: no matches for kind "Deployment" in version "apps/v1beta1"
```

**Comment by Dzhuneyt** [comment]
This happens following the tutorial: https://github.com/jamesward/kubernetes-the-easy-way/blob/master/docs/13-smoke-test.md

**Comment by Dzhuneyt** [comment]
If anyone else is encountering this, I managed to fix it by upgrading to kubectl 1.21 using instructions from: https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/
